app/frontend/testnet/libs/chainlib-wrapper.js

```diff
@@ -0,0 +1,319 @@
+// import * as W from "js-chain-libs/pkg/js_chain_libs_bg.wasm";
```

Comment:
So is this the real wrapper and the two new files later in this PR should be deleted?